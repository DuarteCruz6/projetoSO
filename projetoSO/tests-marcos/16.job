WRITE [(n0,42)(d6,6)(o8,73)(b3,54)]
READ [d6]
DELETE [b3]
SHOW
DELETE [n0]
DELETE [o8,d6]
SHOW
BACKUP
SHOW
WAIT 2000
WRITE [(j0,49)]
READ [j0]
WRITE [(x6,14)(o3,8)(s7,7)]
WAIT 3000
WRITE [(h7,78)(z6,100)(z6,63)(l5,9)(j7,0)]
READ [x6]
WRITE [(q1,83)(r8,70)(x7,88)(s4,43)]
WRITE [(n8,71)(u0,34)(m7,28)(b6,37)(d2,30)]
DELETE [d2,o3,m7]
DELETE [r8,j0]
SHOW
DELETE [s7]
WRITE [(u8,48)]
WRITE [(l9,96)]
READ [x7]
WRITE [(m1,65)(t1,44)(w3,4)(x1,12)(l0,17)]
DELETE [x7,u8,l0]
SHOW
WRITE [(d7,23)]
WRITE [(u8,5)(s7,88)(j2,8)(w2,28)(m2,28)]
WRITE [(i0,40)(q7,21)]
WRITE [(o9,14)(m3,59)]
WRITE [(h1,19)(p8,54)(f7,50)(j6,1)]
READ [p8]
WRITE [(r9,34)(l4,59)]
DELETE [x1]
WRITE [(b8,31)]
SHOW
WRITE [(s9,85)(u1,63)(y8,87)(n0,85)(v3,77)]
WRITE [(y4,74)(n2,40)(k4,96)]
DELETE [w3,h7,j6,m3]
READ [y4]
BACKUP
WRITE [(y2,65)(u5,52)(e2,62)(g3,64)(k1,84)]
READ [k4]
WRITE [(a6,71)(a2,42)]
READ [s7]
WRITE [(x8,93)]
WRITE [(f5,16)(l0,1)]
DELETE [r9]
WRITE [(r4,60)(x3,51)(f2,81)(q9,97)(t5,65)]
DELETE [q7,t1,x3,v3,x6]
WRITE [(b3,78)]
