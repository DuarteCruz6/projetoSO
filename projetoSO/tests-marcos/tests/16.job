WRITE [(k8,15)(d1,42)(a0,6)(m8,65)]
WRITE [(r5,33)(l2,29)]
WRITE [(b3,14)(j4,34)(n7,34)(s2,92)]
WRITE [(c1,50)(w2,0)(l4,96)(t7,22)]
DELETE [l2,d1,b3,s2,n7]
SHOW
WAIT 2000
WRITE [(u2,7)(x0,96)(d9,47)(i0,19)(l6,69)]
WRITE [(r0,28)(x1,8)(u0,69)(e0,29)(z6,27)]
WRITE [(y8,50)]
DELETE [u0,j4]
DELETE [w2,i0,e0,l4]
DELETE [x1]
READ [l6]
READ [x0]
BACKUP
WRITE [(y5,32)(g8,61)(x4,85)]
WRITE [(v4,17)(o3,39)]
READ [r5]
WRITE [(m6,30)]
READ [g8]
WRITE [(a2,95)]
DELETE [t7]
WRITE [(m3,99)(u6,98)(z8,47)(d0,92)]
WRITE [(n6,80)(m2,46)(q4,78)]
WRITE [(e7,1)(s6,34)(e0,16)(e6,97)(g3,29)]
WRITE [(y7,89)(j4,34)(q2,4)(c8,27)(w4,67)]
WRITE [(q9,37)(l3,37)(b5,100)(g6,99)]
WRITE [(b9,92)(e8,40)(i3,57)]
READ [d9]
WRITE [(q3,94)(w8,64)(l7,13)]
WRITE [(v8,40)(j3,55)]
READ [m8]
WRITE [(z7,78)(y4,81)(n1,95)]
WRITE [(v2,54)(c0,99)(j2,27)]
WAIT 3000
WRITE [(o1,78)(f0,2)(h9,24)]
DELETE [i3,c1,q9,d9,g3]
READ [x4]
DELETE [d0]
WRITE [(v7,59)(z2,9)(x8,3)]
READ [l3]
