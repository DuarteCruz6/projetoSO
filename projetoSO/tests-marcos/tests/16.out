(a0, 6)
(c1, 50)
(j4, 34)
(k8, 15)
(l4, 96)
(m8, 65)
(r5, 33)
(t7, 22)
(w2, 0)
[(l6,69)]
[(x0,96)]
[(r5,33)]
[(g8,61)]
[(d9,47)]
[(m8,65)]
[(x4,85)]
[(l3,37)]
